.github/workflows/haskell-ci.yml

```diff
@@ -14,8 +16,17 @@
 #
 name: Haskell-CI
 on:
-  - push
-  - pull_request
+  push: 
+    paths:
+      - '**.hs'
+      - '**.cabal'
+      - 'stack.yaml'
+  # reuse the same filter for pull-requests
+  pull_request:
+    paths:
```

Comment:
Surely we could avoid this duplicity with YAML anchors, right? [Well, no.](https://github.community/t/support-for-yaml-anchors/16128) :disappointed: